### PR TITLE
Changed file name to students/[student_id]/index.tsx. Added option in individual student page to return to student list. Added email functionality to adding students..

### PR DESCRIPTION
**Pull request by hieungo89**
- Corrected file name for student_id folder.
- Added option on individual student page to return to the student list upon clicking.

### CODE REVIEW

File: client/pages/students/[student_id]/index.tsx [hunk 1]
```diff
@@ -16,10 +17,13 @@ const ViewStudentPage = () => {
 
   return (
     <div>
-      <h1>Student {student?.student_id}</h1>
+      <h1>Student: {student?.student_id}</h1>
       <p>
         {student?.first_name} {student?.last_name}
       </p>
+      <button>
+        <Link href={"/students"}>GO BACK</Link>
```
Comment (amantri) [inline client/pages/students/[student_id]/index.tsx:25]: In general back/forward is associated with browser navigation. Maybe change this to "Go to full list" or similar?

File: client/pages/students/[student_id]/index.tsx [hunk 2]
```diff
@@ -16,10 +17,13 @@ const ViewStudentPage = () => {
 
   return (
     <div>
-      <h1>Student {student?.student_id}</h1>
+      <h1>Student: {student?.student_id}</h1>
       <p>
         {student?.first_name} {student?.last_name}
       </p>
+      <button>
```
Comment (codetheweb) [inline client/pages/students/[student_id]/index.tsx:24]: `next/link` doesn't need to be wrapped in a button, it renders a `<a/>`

File: client/pages/students/[student_id]/index.tsx (hunk 1 above)
Comment (hieungo89) [reply client/pages/students/[student_id]/index.tsx:25]: > In general back/forward is associated with browser navigation. Maybe change this to "Go to full list" or similar?

Yes, I can change it to "Return to Student List". Also, I'm not quite sure if the test failure is my portion or not.

Comment (codetheweb) [reply client/pages/students/[student_id]/index.tsx:25]: I believe it was introduced on the push to main.

Open PR to fix: https://github.com/sfbrigade/project-compass/pull/38

File: client/pages/students/[student_id]/index.tsx (hunk 2 above)
Comment (hieungo89) [reply client/pages/students/[student_id]/index.tsx]: I changed it to a <div> for the time being.

File: client/pages/students/[student_id]/index.tsx
[the remark this answers is not in the document]
Comment (amantri) [reply client/pages/students/[student_id]/index.tsx:1]: nit: Can you please change the PR description from "Corrected file name" -> "Changed file name"

**Review by amantri — APPROVED** [review]

File: client/pages/students/[student_id]/index.tsx
[the remark this answers is not in the document]
Comment (hieungo89) [reply client/pages/students/[student_id]/index.tsx:1]: The file name that I changed was the students/[student_id]/index.ts.tsx -> students/[student_id]/index.tsx.

[the remark this answers is not in the document]
Comment (amantri) [reply client/pages/students/[student_id]/index.tsx:1]: Ah ok - sorry - I missed the double suffix. Thanks for fixing it!

File: server/src/db/migrations/1_initial-migrations.sql [hunk 3]
```diff
@@ -11,5 +11,6 @@ CREATE TABLE "student" (
   student_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
   first_name TEXT NOT NULL,
   last_name TEXT NOT NULL,
+  email TEXT NOT NULL UNIQUE,
```
Comment (amantri) [inline server/src/db/migrations/1_initial-migrations.sql:14]: Can you please also update the tests to include this field, and add new test cases (e.g. error handling on duplicate emails)

Comment (hieungo89) [reply server/src/db/migrations/1_initial-migrations.sql:14]: I'll look into it. Thanks for your time, Anup!

**Comment by hieungo89** [comment]
Revised on a different branch with the latest version.

**Pull request closed without merging**